### DISCUSSION TITLE
Why f-droid releases are so delayed?

### DISCUSSION THREAD

**Discussion by Karmalakas** [immich-app/immich#14310] [Q&A] [answered]
It's really annoying to see the orange exclamation mark and not being able to really do anything if the app is installed via f-droid. Usually it's at least three days. Sometimes more than a week. App asks for an update when there is none available. 

Any way to disable this exclamation mark for any updates (app or server)? I'm running Immich on TrueNAS, so server updates also take at least a few days to appear and that orange screaming to my eyes is really frustrating 😞

**Comment by bo0tzz** [comment]
You'd need to ask fdroid, we don't control that build process

**Reply by Karmalakas** [reply]
Fair enough, but still.. Wouldn't it be reasonable to be able to disable this notification? Or at least delay for a week let's say. I wonder how many update the minute when the release comes out 🙄

**Comment by mmomjian** [comment, marked as answer]
disabling in the mobile app has been requested in #2184, please feel free to upvote there. thank you!
